Exercise_D_E/index.html

```diff
@@ -3,14 +3,47 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Flowerbox VI</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="./assets/style/index.css" />
   </head>
   <body>
-    <!-- CODE HERE -->
+    
+    <!-- Nav Bar -->
+    <nav> 
+      <a href="./index.html">
+      <img src="./assets/images/logo.png" alt="flowershop logo">
+      </a>
+    </nav>
 
+    <div class="section1">
+      <img src="assets/images/flowershop.jpg" alt="flowershop" class="flowershop">
+      <div>
+        <h1>Beautiful Flowers</h1>
+        <aside>Learn More</aside>
+      </div>
+    </div>
+    
+  <footer>
+    <div class="grid-container">
+
+        <div class="details">
+          <img src="/FTWD_Web_Day_02/Exercise_D_E/assets/images/flower-icon.png" alt="flower-icon">
```

Comment:
Instead of giving a full path, give a path from where your HTML file is located. For example
`<img src="assets/images/flower-icon.png" alt="flower-icon">`